neovim/__init__.py

```diff
@@ -94,12 +101,13 @@ def attach(session_type, address=None, port=None,
     return Nvim.from_session(session).with_decode(decode)
 
 
-def setup_logging():
+def setup_logging(name):
     """Setup logging according to environment variables."""
     logger = logging.getLogger(__name__)
     if 'NVIM_PYTHON_LOG_FILE' in os.environ:
-        logfile = (os.environ['NVIM_PYTHON_LOG_FILE'].strip() +
-                   '_' + str(os.getpid()))
+        prefix = os.environ['NVIM_PYTHON_LOG_FILE'].strip()
+        major_version = sys.version_info[0]
+        logfile = '{}_py{}_{}'.format(prefix, major_version, name)
```

Comment:
True, I just assumed no one would ever debug python plugins on 2.6. But then let's make it properly, release 0.1.10 with the HEADSUP that it is the last version supporting 2.6, and then formally remove 2.6 support.